libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -131,6 +131,13 @@ const AP_Param::GroupInfo SoaringController::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("MAX_RADIUS", 17, SoaringController, max_radius, -1),
 
+    // @Param: THML_BANK
+    // @DisplayName: Thermalling bank angle
+    // @Description: This parameter sets the bank angle to use when thermalling. Typically 30 - 45 degrees works well.
+    // @Range: 30 45
```

Comment:
seems to be pretty narrow range for bank...is it really this restricted?

Reply:
This is the recommended range, as always you can set outside this range by accepting the warning. Note also that this is the average bank, so less or more bank up to LIM_ROLL_CD can be used for centering corrections

Reply:
"Roll" is used more in this library than "bank" so lets stick with that terminology

Reply:
although... there's NAVL1_LIM_BANK, which this is overriding so lets name this SOAR_LIM_BANK and in the description mention that it overrides NAVL1_LIM_BANK and if this is new param is set to 0 then it uses NAVL1_LIM_BANK.

Reply:
- Widened the limits to 20 - 50deg.
- Kept the naming as-is since it's not a limit but rather a target. Removed misleading left-over comment in mode_thermal that I expect was causing confusion